src/webots/maths/WbImage.hpp

```diff
@@ -24,8 +24,6 @@ class WbImage {
   int width() const { return mWidth; }
   int height() const { return mHeight; }
 
-  WbImage *downscale(int width, int height, int xBlurRadius, int yBlurRadius);
-
```

Comment:
Can't we get rid of that class? It seems pretty useless now.

Reply:
Indeed